README.md

```diff
@@ -9,92 +9,388 @@
   }
 }
+```
+
+The `handleChangeCardTitle` itself would actually "delegate" itself to its
+parent component, in this case the actual `<Board />` component:
 
-// stores/togglerStore.js
-const Store = require('./Store')
+```js
+  handleChangeCardTitle (cardIndex, ev) {
+    this.props.onChangeCardTitle(this.props.id, cardIndex, ev)
+  }
+```
+
+The board would delegate to **its** parent, which is the actual `<App />`
+component:
+
+```js
+  handleChangeCardTitle (columnIndex, cardIndex, ev) {
+    this.props.onChangeCardTitle(columnIndex, cardIndex, ev)
+  }
+```
 
-const initialState = {
-  on: false
+The `<App />` component itself would now **actually** update the underlying
+state:
+
+```js
+class App extends React.Component {
+  constructor (props) {
+    super(props);
+    this.state = { board: [] };
+  }
+  componentDidMount () {
+    fetchBoard().then(board => this.setState({ board }));
+  }
+  // Magic comes in here:
+  handleChangeCardTitle (columnIndex, cardIndex, ev) {
+    // Actually we wouldn't even want to mutate this.state directly. Ideally we
+    // should make a copy of the respective card and apply our change there.
+    this.state.board[columnIndex][cardIndex].title = ev.target.value;
+    updateBoard(this.state).then(() => {
+      // Do some more magic here.
+    });
+  }
+  render () {
+    const {board} = this.state;
+    return <Board board={board} onChangeCardTitle={this.hanldeChangeCardTitle(ev)} />;
+  }
 }
+```
+
+### Change Listeners? — We don't need them!
+
+Now this _would_ work, but it's complicated beyond measures. It's much easier
+and less error-prone to move our state out into a separate store.
+
+```
+<App /> (can update this.state.board)
+  <Board /> (has to delegate to parent component)
+    <Column /> (has to delegate to parent component)
+      <Card /> (has to delegate to parent component)
+      <Card /> (has to delegate to parent component)
+    <Column /> (has to delegate to parent component)
+      <Card /> (has to delegate to parent component)
+```
+
+The store could then handle the card updates in one form or another. The
+`<App />` component could subscribe to the store and all components would 
+happy! No needless event handlers, just one, flat state tree.
+
+But let's take a step back first and summarize why the above solution is
+problematic:
+
+1. We had to add a separate event handler on each level.
+2. Needless redundancy: `<Column />` and `<Board />` share _almost_ the same
+   handler function — **almost**.
+3. Adding a separate component "in-between" our existing components would 
+   complicated beyond measure — just imagine adding a separate
+  `<BoardVersion />` component as a child of `<Board />`.
+
+Clearly moving the state out into a separate, completely isolated store is the
+desired solution here. Instead of communicating via components that "pass
+through" events, we directly update the store (at least for now) and render
+subsequent changes by passing down updated `props`.
+```
```

Comment:
These backticks don't need to be here, they're causing the Markdown to render as code below